kafka-to-kv/function.yaml.values

```diff
@@ -0,0 +1,70 @@
+InitialOffset:
+  displayName: Initial Offset
+  kind: choice
+  description: "Kafka initial offset"
+  required: true
+  attributes:
+    choices: [earliest, latest]
+    defaultValue: latest
+
+ConsumerGroup:
+  displayName: Consumer Group
+  kind: string
+  description: "Kafka consumer group name"
+  required: false
+
+Topic:
+  displayName: Topic
+  kind: string
+  description: "Kafka topic name"
+  required: true
+
+Broker:
+  displayName: Kafka URL and port
+  kind: string
+  description: "Kafka broker address"
+  required: true
+
+NginxHost:
```

Comment:
@sahare92 need to change all mentions of `nginx` to `V3ioApiEndpoint` (nginx is an internal and very unclear term)